lib/json_info_codec.dart

```diff
@@ -294,7 +295,7 @@ class AllInfoToJsonConverter extends Converter<AllInfo, Map>
   Map convert(AllInfo info) => info.accept(this);
 
   Map _visitList(List<Info> infos) {
-    var map = <String, Map>{};
+    var map = new SplayTreeMap<String, Map>(compareNatural);
```

Comment:
Added a comment.

You want the actual IDs to stay stable. So you don't churn the lines like crazy. Churned lines make scanning the diff next to impossible.